docs/source/examples.md

```diff
@@ -11,6 +11,8 @@
 
 [Cifar10 optimization with OpenVINO for Intel HW](https://github.com/microsoft/Olive/tree/main/examples/cifar10_openvino_intel_hw)
 
+[GPT-J optimization with Intel® Neural Compressor Post Training Quantization](https://github.com/microsoft/Olive/tree/main/examples/gptj)
```

Comment:
for the document update, need to skip the link in first PR since the code is not in main yet. 
then create a second PR to update the link.